photon_installer/installer.py

```diff
@@ -88,12 +89,12 @@ class Installer(object):
     linux_dependencies = ["devel", "drivers", "docs", "oprofile", "dtb"]
 
     def __init__(self, working_directory="/mnt/photon-root",
```

Comment:
All these default locations should be stored in variables. Don't hard code them.
If we change it in one place, it should reflect in all places.

Reply:
+1